### DISCUSSION TITLE
My Notes Field with Worked Station

### DISCUSSION THREAD

**Discussion by dimarmnes** [foldynl/QLog#382] [Q&A] [answered]
I started to notice now that when I contact a worked station, Qlog fills the My Notes field with the data from the last record of that station. There is a reason for that? when it is supposed to be data from the QSO and not from the station.

DE LU1IDC Diego

**Comment by foldynl** [comment, marked as answer]
It is not a bug, it is a feature. you can continue your notes like this. It was requested.

**Reply by dimarmnes** [reply]
Thanks for the reply, I'll take that into account.
